services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -273,6 +274,15 @@ export class WorkflowService {
     return updateResult;
   }
 
+  async assignWorkflowToAssignee(workflowRuntimeId: string, data: WorkflowWhereAssignInput) {
+    const updatedWorkflowRuntime = await this.workflowRuntimeDataRepository.updateById(
+      workflowRuntimeId,
+      { data: data },
```

Comment:
If the scope of the update is the assignee why we get the whole data and not just the assignee, and explicitly changing only this column?

Less room for errors

Reply:
the class validator of WorkflowWhereAssignInput validates the data being updated.
do you think it's preferable to 
{ data: { assigneeId: data.assigneeId }? even though WorkflowWhereAssignInput formats the updated data?

Reply:
I dont have to pass WorkflowWhereAssignInput, you could just pass `assgineeId` - I think the Input should stay in the controller level - other users of this functions only cares about the assginee id and should not be aware of other structure